tests/publiccloud/download_repos.pm

```diff
@@ -33,6 +33,10 @@ sub run {
         record_info('Skip download', 'Skipping maintenance update download (triggered by setting)');
         return;
     } else {
+        # Remove the ~/repos so they can be redownloaded
+        if (get_var('PUBLIC_CLOUD_REDOWNLOAD_MU')) {
+            script_run("rm -rf ~/repos");
```

Comment:
why not assert?

Reply:
I was not 100% but only 99% sure that the `rm -rf` exits `0` if the target does not exist.

Reply:
`-f` forces `0` result if the dir/file doesn't exist:
```
cat nothing
cat: nothing: No such file or directory

rm -f nothing
echo $?
0
```